Content.Shared/Blocking/BlockingSystem.cs

```diff
@@ -135,8 +137,8 @@ public bool StartBlocking(EntityUid item, BlockingComponent component, EntityUid
 
         if (component.BlockingToggleAction != null)
         {
-            //Don't allow someone to block if they're in a container.
-            if (_containerSystem.IsEntityInContainer(user) || !_mapManager.TryFindGridAt(xform.MapPosition, out var grid))
+            //Don't allow someone to block if they're in a container or if they're buckled.
```

Comment:
doesnt buckling/containers both just parent the entity? wouldnt it be reasonable to instead just check if they aren't parented to a grid, and disallow blocking in that case

Reply:
Good catch, thanks!